i18n-embed/src/fluent.rs

```diff
@@ -257,6 +257,19 @@ impl FluentLanguageLoader {
 
         (closure)(&mut iter)
     }
+
+    /// Set whether the underlying Fluent logic should insert Unicode
+    /// Directionality Isolation Marks around placeables.
+    ///
+    /// See [`fluent::bundle::FluentBundleBase::set_use_isolating`] for more
+    /// information.
+    ///
+    /// Default: `true`.
+    pub fn set_use_isolating(&self, value: bool) {
+        for bundle in self.language_config.write().language_bundles.as_mut_slice() {
```

Comment:
Rust surprised me here. I'm able to get mutable access to the a nested field several levels down, and yet we're borrowing the parent immutably with `&self`.

And a question with regard to `write()`: I see that it blocks the thread. Is my overall approach to getting down to the internal `bundle` a good one?

Reply:
`language_config` is wrapped in an [RwLock](https://doc.rust-lang.org/std/sync/struct.RwLock.html) which gives it safe interior mutability with the use of runtime checks. The downside in this case is that threads will block if they try to access it when someone holds the write lock to it, but I hoped that it would not be common to be making changes to the `language_config` during a performance critical piece of code, thus it would at least be better than using a [Mutex](https://doc.rust-lang.org/std/sync/struct.Mutex.html). It also has the property that the write lock cannot be obtained until all the read locks have been dropped. 

I guess we need to be careful not to introduce a deadlock if this method will be called within another method that already holds the lock, seems like that's not the case here so it looks good to me.